ocw/lib/gce.py

```diff
@@ -89,115 +90,33 @@ def delete_instance(self, instance_id, zone):
     def url_to_name(url):
         return url[url.rindex("/")+1:]
 
-    def parse_image_name(self, img_name):
-        regexes = [
-            # sles12-sp5-gce-x8664-0-9-1-byos-build1-56
-            re.compile(
-                r"""^sles
-                    (?P<version>\d+(-sp\d+)?)
-                    -
-                    (?P<flavor>gce)
-                    -
-                    (?P<arch>[^-]+)
-                    -
-                    (?P<kiwi>\d+-\d+-\d+)
-                    -
-                    (?P<type>(byos|on-demand))
-                    -build
-                    (?P<build>\d+-\d+)
-                    """,
-                re.RegexFlag.X,
-            ),
-            # sles15-sp2-byos-x8664-0-9-3-gce-build1-10
-            # sles15-sp2-x8664-0-9-3-gce-build1-10
-            re.compile(
-                r"""^sles
-                    (?P<version>\d+(-sp\d+)?)
-                    (-(?P<type>[-\w]+))?
-                    -
-                    (?P<arch>[^-]+)
-                    -
-                    (?P<kiwi>\d+-\d+-\d+)
-                    -
-                    (?P<flavor>gce)
-                    -
-                    build
-                    (?P<build>\d+-\d+)
-                    """,
-                re.RegexFlag.X,
-            ),
-            # sles15-sp1-gce-byos-x8664-1-0-5-build1-101
-            re.compile(
-                r"""^sles
-                (?P<version>\d+(-sp\d+)?)
-                (-(?P<flavor>gce))?
-                -
-                (?P<type>[-\w]+)
-                -
-                (?P<arch>[^-]+)
-                -
-                (?P<kiwi>\d+-\d+-\d+)
-                -
-                build
-                (?P<build>\d+-\d+)
-                """,
-                re.RegexFlag.X,
-            ),
-        ]
-        return self.parse_image_name_helper(img_name, regexes)
-
     def cleanup_all(self):
-        images = list()
         request = self.compute_client().images().list(project=self.__project)
         while request is not None:
             response = request.execute()
             if "items" not in response:
                 break
             for image in response["items"]:
-                # creation:2019-11-04T14:23:06.372-08:00
-                # name:sles12-sp5-gce-x8664-0-9-1-byos-build1-56
-                m = self.parse_image_name(image["name"])
-                if m:
-                    images.append(
-                        Image(
-                            image["name"],
-                            flavor=m["key"],
-                            build=m["build"],
-                            date=parse(image["creationTimestamp"]),
+                if self.older_than_max_age_hours(parse(image["creationTimestamp"]).astimezone(timezone.utc)):
```

Comment:
```suggestion
                if self.is_outdated(parse(image["creationTimestamp"]).astimezone(timezone.utc)):
```
More descriptive (and shorter!) method name suggestion

Reply:
done